app/Traits/Post.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Traits;
+
+use Illuminate\Support\Facades\Storage;
+
+trait Post
+{
+    public function handleImageOld($imageNew, $imageOld, $arrImageDeleted)
+    {
+        $imageDeleted = [];
```

Comment:
var_dump cho a casi $this xem nó là object gì?

Sau đó, nếu $this có đủ các cái imageOld thì xem xét ko cần truyền imageOld trong param của function này nữa.